Examples/ServiceExamples/Scripts/ExampleDiscovery.cs

```diff
@@ -83,6 +83,12 @@ public class ExampleDiscovery : MonoBehaviour
     private bool _readyToContinue = false;
     private float _waitTime = 10f;
 
+    private bool _listCredentialsTested = false;
```

Comment:
just a minor thing but was thinking we should have variable names consistent. i.e, `_getCredentialTested` should be like others i.e, `_getCredentialsTested`